tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/Constant.java

```diff
@@ -36,19 +36,37 @@
  */
 public class Constant<T extends TType> extends BaseInitializer<T> {
 
-  private final Number numberValue;
-  private final Boolean booleanValue;
+  private final double doubleValue;
+  private final long longValue;
+  private final boolean booleanValue;
+  private final ValueType valueType;
 
   /**
    * Creates an Initializer that generates tensors with a constant value.
    *
    * @param tf the TensorFlow Ops
-   * @param value a number value
+   * @param value a double value
```

Comment:
The javadoc says double, but the constructor expects a long.

Reply:
OK